### DISCUSSION TITLE
Link inside textarea

### DISCUSSION THREAD

**Discussion by anedosugov** [vuetifyjs/vuetify#11928] [General] [answered]
How can we render a clickable link inside a textfield or textarea?

**Comment by kamalsprasad** [comment, marked as answer]
You'll some sort of rtf editor. There's [quill](https://github.com/surmon-china/vue-quill-editor) and [tiptap](https://tiptap.dev/). There's vuetify integration for tiptap here: https://github.com/iliyaZelenko/tiptap-vuetify
